sources/api/migration/migrations/v1.19.3/aws-admin-container-v0-11-6/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "aws-admin-container-v0-11-6"
+version = "0.1.0"
+authors = ["Yutong Sun <yutongsu@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+migration-helpers = { path = "../../../migration-helpers", version = "0.1.0"}
```

Comment:
Please configure your editor to save files with a newline at the end.

For VS Code there is this:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

(Note, this is a different behavior than manually adding a blank line at the end of the file - we don't want to do that.)

Reply:
All of the new `Cargo.toml` files have the missing-newline problem.

Reply:
Thanks for the link, updated the commits with the newline setup.